recipes/anaconda-project/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "anaconda-project" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/Anaconda-Platform/anaconda-project.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  entry_points:
+    - anaconda-project = anaconda_project.cli:main
+
+requirements:
+  build:
+    - anaconda-client
+    - backports_abc >=0.4  # [py2k]
+    - psutil
+    - python
+    - requests
+    - ruamel_yaml
+    - setuptools
+    - singledispatch
+    - tornado
```

Comment:
Would add that there are a few of these missing from `run`. Might want to double check to make sure this is correct. Not knowing much about how this built, I'll simply trust your judgement if you say this is fine.